### ISSUE TITLE
Inserter: IE11 and JAWS don't announce the Inserter blocks

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5827]
Tested on Windows 10 with Internet Explorer 11 and JAWS 2018:

When navigating through the available block types in the Inserter, IE11 and JAWS 2018 don't announce anything. Resounding silence. Works with Firefox ESR and JAWS 2018 so it mus tbe something specific to the combo IE11 + JAWS 2018.

Luckily, repeating the button text as value of an aria-label attribute, fixes it. PR incoming.

Note: I have no idea why this happens. I've inspected the accessibility tree and the accessible name is populated correctly so it should be properly exposed to browsers. My only guess is that the SVG icon as first children of the menu items plays a role, as it's technically "empty", and this specific combo might have troubles with an empty first children.
